nemo/nemo/backends/pytorch/nm.py

```diff
@@ -209,29 +209,29 @@ def input_ports(self):
         return {}
 
     def get_weights(self):
-        logging.warning(
+        nemo.logging.warning(
```

Comment:
question to @okuchaiev : DataLayers/NonTrainableNMs do not have weights by definition, do we have to log "No blablabla" warnings, as DO_NOTHING is both the default AND desired behavior?

maybe we can simply remove all those warnings?

Reply:
I'd say remove this warning for now.

Reply:
Commented out all warnings from DataLayerNM and LossNM

Reply:
perfect, thank you! 👍